src/gui/torrentcontentmodelfile.cpp

```diff
@@ -33,42 +33,48 @@
 
 TorrentContentModelFile::TorrentContentModelFile(const QString &fileName, qulonglong fileSize,
                                                  TorrentContentModelFolder* parent, int file_index)
-  : TorrentContentModelItem(parent)
-  , m_fileIndex(file_index)
+    : TorrentContentModelItem(parent)
+    , m_fileIndex(file_index)
 {
-  Q_ASSERT(parent);
+    Q_ASSERT(parent);
 
-  m_name = fileName;
+    m_name = fileName;
 
-  // Do not display incomplete extensions
-  if (m_name.endsWith(".!qB"))
-    m_name.chop(4);
+    // Do not display incomplete extensions
+    if (m_name.endsWith(".!qB"))
+        m_name.chop(4);
 
-  m_size = fileSize;
+    m_size = fileSize;
 }
 
 int TorrentContentModelFile::fileIndex() const
 {
-  return m_fileIndex;
+    return m_fileIndex;
 }
 
 void TorrentContentModelFile::setPriority(int new_prio, bool update_parent)
```

Comment:
Fix the coding style is not to spoil the story. This is the same change as any other (for example, as adding a new feature or bugfix). So either do it or don't do it at all.